crates/rspack_testing/src/run_fixture.rs

```diff
@@ -37,7 +37,10 @@ pub async fn test_fixture_js(fixture_path: &Path) -> Compiler<AsyncNativeFileSys
 }
 #[tokio::main]
 pub async fn test_fixture_css(fixture_path: &Path) -> Compiler<AsyncNativeFileSystem> {
-  test_fixture_share(fixture_path, &|s| s.ends_with(".css")).await
+  test_fixture_share(fixture_path, &|s| {
+    s.ends_with(".css") || (s.ends_with(".js") && !s.contains("runtime.js"))
```

Comment:
Maybe we should create a new one exclusively for css module test instead of changing this. I will optimize this once this PR is merged.